### PR TITLE
Updated Footer

### LINKED ISSUE #391: Footer
Similar to ZotHacks 2024: footer needs links to
- Hack (https://hack.ics.uci.edu)
- Email (irvinehacks2025@gmail.com)
- Discord (https://discord.com/invite/pvkGxq2AWM)
- Instagram (https://www.instagram.com/hackatuci/)
- Tiktok (https://www.tiktok.com/@hackatuci)

with its UI updated using [Figma](https://www.figma.com/design/CyxJ0OSjgNPwb0eeTgq84s/IrvineHacks-2025?node-id=1-2&node-type=frame&m=dev) as reference

### PR DESCRIPTION
**Pull request by cyriljoby**
Resolves #391 

Created footer with icons and links to Hack's instagram, tiktok, mail and discord.
Updated the heart icon to reflect this year's design.

### CODE REVIEW

File: apps/site/src/lib/components/Footer/Footer.module.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .footer {
 	color: white;
-	background-color: #432810;
+	// background-color: #432810;
```
Comment (waalbert) [inline apps/site/src/lib/components/Footer/Footer.module.scss:3]: Chore: remove unused comment

File: apps/site/src/lib/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -50,15 +51,17 @@ const Footer = () => {
 		<div className="flex flex-row gap-1 items-center mt-2 max-[600px]:flex-col max-[210px]:gap-0 max-[210px]:mt-0">
 			<div className="flex flex-row gap-1 items-center max-[210px]:flex-col max-[210px]:gap-0">
 				<p className="m-0">Made with</p>
-				<svg className=" fill-white" xmlns="http://www.w3.org/2000/svg" height="16" width="16" viewBox="0 0 512 512"><path d="M47.6 300.4L228.3 469.1c7.5 7 17.4 10.9 27.7 10.9s20.2-3.9 27.7-10.9L464.4 300.4c30.4-28.3 47.6-68 47.6-109.5v-5.8c0-69.9-50.5-129.5-119.4-141C347 36.5 300.6 51.4 268 84L256 96 244 84c-32.6-32.6-79-47.5-124.6-39.9C50.5 55.6 0 115.2 0 185.1v5.8c0 41.5 17.2 81.2 47.6 109.5z"/></svg>
+				<Image src={redHeart} alt="Red heart icon" layout="intrinsic" />
+
 				<p className="m-0">in Irvine, CA <span className="max-[600px]:hidden">•</span></p>
```
Comment (waalbert) [inline apps/site/src/lib/components/Footer/Footer.tsx:56]: Due to Student Center regulations, we have to specify that IrvineHacks is hosted by us, Hack at UCI, so switch the text of the footer to say "Made with ❤️ by Hack at UCI ..."

And that means we also have to add our Hack logo linked to our club site.

**Review by waalbert — CHANGES_REQUESTED** [review]
Awesome work Cyril! Here are a few requested changes

File: apps/site/src/assets/logos/discord.svg
Comment (taesungh) [inline apps/site/src/assets/logos/discord.svg:1]: This does not appear to be the official Discord logo. Please make sure to use the official version (e.g. `discord-mark-white.svg`) from [Discord's Branding Guidelines](https://discord.com/branding).

File: apps/site/src/assets/logos/tiktok.svg
Comment (taesungh) [inline apps/site/src/assets/logos/tiktok.svg:1]: Same note here: this does not appear to be an official TikTok logo, please use `TikTok-Social-Icon-Rounded-Square-Mono-Black.svg` from the [TikTok Brand Guidelines](https://tiktokbrandbook.com).

File: apps/site/src/lib/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,64 +1,81 @@
...
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						<span style={{ color: "#FBA80A" }}>Hack @ UCI Team</span>
+					</a>
+				</p>
+			</div>
+		</footer>
 	);
+	// ;#2878A6
```
Comment (waalbert) [inline apps/site/src/lib/components/Footer/Footer.tsx:78]: Chore: remove unused comment

**Review by waalbert — CHANGES_REQUESTED** [review]
Some more things

File: apps/site/src/lib/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -1,64 +1,81 @@
...
-		"alt": "Hack Logo that links to Hack at UCI's website",
-		"link": "https://hack.ics.uci.edu"
+		logo: hack,
+		alt: "Hack logo that links to Hack at UCI's website",
+		link: "https://hack.ics.uci.edu/",
+	},
+	{
+		logo: mail,
+		alt: "Email that directs to hack@uci.edu",
+		link: "mailto:hack@uci.edu",
```
[the remark this answers is not in the document]
Comment (taesungh) [reply apps/site/src/lib/components/Footer/Footer.tsx:19]: @waalbert, could you set up forwarding on a vanity email instead, e.g. contact@irvinehacks.com ?

[the remark this answers is not in the document]
Comment (waalbert) [reply apps/site/src/lib/components/Footer/Footer.tsx:19]: Yes, we will use the vanity email instead.

**Review by waalbert — DISMISSED** [review]
Thanks for the work Cyril! Don't merge until the new Landing Page is completed

**Review by waalbert — APPROVED** [review]

**Pull request merged**